components/organisms/TopicForm.tsx

```diff
@@ -69,9 +70,18 @@ const useStyles = makeStyles((theme) => ({
   localVideo: {
     width: "100%",
   },
-  videoButtons: {
+  videoController: {
+    display: "flex",
+    justifyContent: "space-between",
```

Comment:
左右に離して配置する事自体は再生開始と再生終了をレイアウトによって表現できておりよいと思います
幅いっぱい離れると画面幅によっては操作にストレスを感じる距離になりうる気がします
space-aroundである程度中央寄せするのはいかがでしょう

```suggestion
    justifyContent: "space-around",
```

Reply:
わからないですがそうかもしれないですね。やってみました。